scripts/utils/trading_strategy_helpers.js

```diff
@@ -154,11 +113,12 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
 
   const globalTokenPromisesFromId = {}
   /**
-   * Queries EVM for ERC20 token details by token id
-   * and returns a list of detailed token information.
+   * Queries EVM for ERC20 token details by token id and returns a list of detailed token information.
+   *
    * @param {SmartContract} exchange BatchExchange, contract, or any contract implementing `tokenIdToAddressMap`
-   * @param {integer[]} tokenIds list of *unique* token ids whose data is to be fetch from EVM
-   * @return {Promise<TokenObject>[]} list of detailed/relevant token information
+   * @param {number[]} tokenIds list of *unique* token ids whose data is to be fetch from EVM
+   * @param {boolean} [debug=false] prints log statements when true
+   * @returns {Promise<TokenObject>[]} list of detailed/relevant token information
```

Comment:
Same issue as before.
```suggestion
   * @returns {object} object mapping token ids to a promise of relevant token information
```